yt_dlp/extractor/nitter.py

```diff
@@ -108,32 +168,33 @@ def _real_extract(self, url):
                 'ext': ext
             }]
 
-        title = (
-            self._og_search_description(webpage).replace('\n', ' ')
-            or self._html_search_regex(r'<div class="tweet-content[^>]+>([^<]+)</div>', webpage, 'title'))
+        title = self._og_search_description(full_webpage)
+        if not title:
+            title = self._html_search_regex(r'<div class="tweet-content[^>]+>([^<]+)</div>', webpage, 'title')
```

Comment:
Is it not better to give priority to `tweet-content` since the `description` is of the whole webpage?

Also, I don't understand why `description = title`, but I wont ask u to change that since that was how it was already